apps/cli/src/commands/run.ts

```diff
@@ -99,6 +100,16 @@ export const handleRunCliCommand = async (
   const nameOrPath = String(args._.at(-1));
 
   if (existsSync(nameOrPath)) {
+    try {
+      if (await isWorkflowFile(nameOrPath)) {
+        await runWorkflowFile(nameOrPath);
+        return;
+      }
+    } catch (e: any) {
+      console.error(e);
+      console.log(e?.stack);
+      return;
+    }
```

Comment:
since you have added boilerplate to run codemods that have `engine: workflow`, can this be removed?